src/renderer/Metadata.js

```diff
@@ -18,6 +18,7 @@ export default class Metadata {
         this.categoryToID = new Map();
         this.IDToCategory = new Map();
         this.numCategories = 0;
+        this.extent = extent || 4096;
```

Comment:
Maybe we could extract 4096 to its own explaining constant (as in windshaft.js), using `DEFAULT_MVT_EXTENT`

Reply:
Done. Thx!